### ISSUE TITLE
TestsOutputConsolePrinter throws NullPointerException

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#315]
```
What steps will reproduce the problem?
1. Write simple TestCase with console.log.
2. Disconnect remote test browser connect by closing window.
2. See the error.

What is the expected output? What do you see instead?
Proper log message.

Currently just error.

What version of the product are you using? On what operating system?
JsTestDriver-1.3.3d
IDE: Intellij PhpStorm 3.0 (Build #PS-111.19).
OS: Mac OS X 10.7.2.
Java: 1.6
```

Original issue reported on code.google.com by `actuosus` on 25 Dec 2011 at 6:10

Attachments:
- [Stacktrace.log](https://storage.googleapis.com/google-code-attachments/js-test-driver/issue-315/comment-0/Stacktrace.log)

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 26 Dec 2011 at 5:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 26 Dec 2011 at 5:28
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.4 version of the plugin.
```

Original comment by `Sergey.S...@gmail.com` on 23 Feb 2012 at 6:01
- Changed state: **Fixed**
